lib/web_ui/test/dom_renderer_test.dart

```diff
@@ -17,41 +17,26 @@ void main() {
 
 void testMain() {
   test('populates flt-renderer and flt-build-mode', () {
-    DomRenderer();
+    FlutterViewEmbedder();
     expect(html.document.body!.attributes['flt-renderer'],
         'html (requested explicitly)');
     expect(html.document.body!.attributes['flt-build-mode'], 'debug');
   });
 
-  test('creating elements works', () {
-    final DomRenderer renderer = DomRenderer();
-    final html.Element element = renderer.createElement('div');
-    expect(element, isNotNull);
-  });
-
   test('can set style properties on elements', () {
-    final DomRenderer renderer = DomRenderer();
-    final html.Element element = renderer.createElement('div');
-    DomRenderer.setElementStyle(element, 'color', 'red');
+    final html.Element element = html.document.createElement('div');
+    setElementStyle(element, 'color', 'red');
```

Comment:
Small nit:

Maybe these tests should be moved to `test/engine/util_test.dart`, now that they live in a separate file.

Also, should this file be renamed as `test/engine/embedder_test.dart` so it matches the rename of the file it tests?

Reply:
Done.